packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/upgrade-request/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,13 @@
+{
+  "dedicated_server_dashboard_upgrade_title_ram": "Augmenter la RAM",
+  "dedicated_server_dashboard_upgrade_title_storage": "Augmenter le stockage des données",
+  "dedicated_server_dashboard_upgrade_submit_label": "Confirmer",
+  "dedicated_server_dashboard_upgrade_cancel_label": "Annuler",
+  "dedicated_server_dashboard_upgrade_content_ram": "Vous avez besoin de davantage d’espace et vous souhaiter augmenter la RAM de votre serveur <strong>{{ projectId }}</strong> ?",
+  "dedicated_server_dashboard_upgrade_content_storage": "Vous avez besoin de davantage d’espace et vous souhaiter augmenter le stockage de votre serveur <strong>{{ projectId }}</strong> ?",
+  "dedicated_server_dashboard_upgrade_label": "Décrivez votre demande : créneau souhaité pour l’intervention, volume demandé pour les prochains mois, disques à ajouter, de ram…",
```

Comment:
`de ram` ? I'm having trouble understanding that part of the sentence

Reply:
i have take this sentence to the spec, i have ping Marion for a re-check.

Reply:
has been updated `Vous avez besoin de davantage de mémoire vive et vous souhaitez augmenter la RAM de votre serveur [project_ID].`

Reply:
@Heiiji I can't see the content update

Reply:
is change again after a recheck of QA and Marion : "Vous avez besoin de plus de mémoire vive et vous souhaitez augmenter la RAM de votre serveur ".